deepinv/tests/test_external_libraries.py

```diff
@@ -0,0 +1,111 @@
+import deepinv as dinv
+import torch
+import pytest
+
+import unittest.mock as mock
+
+
+class TestTomographyWithAstra:
+    def dummy_compute_norm(cls, x0: torch.Tensor) -> torch.Tensor:
+        return torch.tensor(1.0).to(x0)
+
+    def dummy_projection(cls, x: torch.Tensor, out: torch.Tensor) -> None:
+        out[:] = 1.0
+
+    @pytest.mark.parametrize(
+        "is_2d,geometry_type,normalize",
+        [
+            (True, "parallel", False),
+            (True, "parallel", True),
+            (True, "fanbeam", False),
+            (True, "fanbeam", True),
+            (False, "parallel", False),
+            (False, "parallel", True),
+            (False, "conebeam", False),
+            (False, "conebeam", True),
+        ],
+    )
+    @mock.patch.object(
```

Comment:
I like this test! But do you think it would make more sense to not repeat code? One solution would be to merge it with the other tomography test and instead have an environment variable DEEPINV_MOCK_EXTERNAL or something like we have for mocking datasets in #490, and if it's true we run the mocked test